### DISCUSSION TITLE
idea: generate from spec

### DISCUSSION THREAD

**Discussion by a-h** [AccentDesign/gcss#5] [Ideas]
One of the things I really like about https://github.com/delaneyj/gostar is that it generates code direct from the spec.

I'm not sure how and if gcss covers all CSS properties, and how much effort it would be to keep up to date. I thought that generating from specs might be something you'd be interested in.

If you generated from spec, then it would be clear how much of the CSS spec was/wasn't covered.

Just an idea!

**Comment by stuartaccent** [comment]
I like the idea of it, the props at the minute are a combination of me trying it out with a wails project we are working on and a bit of that looks like summit we would need. So sparse at best.

originally this was only ever a kind of bugger about with idea and see where it goes if anywhere at all. I suspect all the stars over the last few days are more of a result of a reddit post and just the shear volume of traffic the repo got since sat. So im not conviced there is any real desire for it as yet if ever. The idea of it has still only partially convinced me of its value if at all. There are a hell of a lot of holes and things that just feel like I can't be bothered with.

So I will have a look, and maybe just see what the next few months bring. It may be that something else comes out if it and that this is only a stepping stonne to something better.

Stu
